dashboard/app/reporting_email.go

```diff
@@ -388,10 +388,23 @@ func handleTestCommand(c context.Context, info *bugInfoResult, msg *email.Email)
 		return replyTo(c, msg, info.bugReporting.ID,
 			fmt.Sprintf("want 2 args (repo, branch), got %v", len(args)))
 	}
-	reply := handleTestRequest(c, &testReqArgs{
+	reply := ""
+	err := handleTestRequest(c, &testReqArgs{
 		bug: info.bug, bugKey: info.bugKey, bugReporting: info.bugReporting,
 		user: msg.Author, extID: msg.MessageID, link: msg.Link,
 		patch: msg.Patch, repo: args[0], branch: args[1], jobCC: msg.Cc})
+	if err != nil {
+		log.Errorf(c, "failed to handle the test request: %v", err)
```

Comment:
We should not log as error BadTestRequestError. It's an invalid user request, but log.Errorf shows up on admin page an application error which we need to fix.